### DISCUSSION TITLE
is it possible to define starting point for optimization?

### DISCUSSION THREAD

**Discussion by rczerminski-valo** [coin-or/rbfopt#41] [Ideas]
Is it possible to define starting point for optimization in addition to var_lower, var_upper in RbfoptUserBlackBox?
I do not see it as an option right now.

**Comment by gnannicini** [comment]
You can give a list of points that you want to be evaluated as part of the initialization (see the argument init_node_pos for RbfoptAlgorithm). The algorithm is not an iterative local algorithm, therefore it does not require a starting point like, e.g., gradient descent.
